src/app/routes/gateway.js

```diff
@@ -94,4 +94,23 @@ module.exports = (Router, Service) => {
         res.status(500).send({ error: err.message });
       });
   });
+
+  Router.get('/gateway/registerCompleted', basicAuth, (req, res) => {
+    const email = req.query.email;
+
+    return Service.User.FindUserByEmail(email)
+      .then((user) => {
+        const { uuid, registerCompleted } = user;
+
+        res.status(200).send({
```

Comment:
You could return everything and get what you want on the other side as this will be a standard GET. Thus, the required data can be reused or extended on the other side of the gateway.